pkgs/tools/admin/acme.sh/default.nix

```diff
@@ -16,7 +16,9 @@ stdenv.mkDerivation rec {
     mkdir -p $out $out/bin $out/libexec
     cp -R $src/* $_
     makeWrapper $out/libexec/acme.sh $out/bin/acme.sh \
-      --prefix PATH : "${lib.makeBinPath [ socat openssl curl iproute ]}"
+      --prefix PATH : "${lib.makeBinPath (
+        [ socat openssl curl ] ++ lib.optional stdenv.isLinux iproute
```

Comment:
Can you instead use "unixtools.netstat" here? This is provided for both linux and darwin.

Reply:
Looking at the [original code][], it prefers `ss` if available, and falls back to sniffing for netstat versions by parsing help strings. In this case I would prefer to provide `ss` via iproute if available, and netstat otherwise. (PR updated)

[original code]: https://github.com/Neilpang/acme.sh/blob/f62a4a0c0ccf6cd73b5746dd8b8790ce3c512833/acme.sh#L1182-L1214